### PR TITLE
[quest] Mark "The Thunderdrome!" as breadcrumb quest

### PR DESCRIPTION
**Pull request by TorpedoCXC**
Marked as breadcrumb quest for 25067 Thunderdrome: The Ginormus!

### CODE REVIEW

**Comment by BreakBB** [comment]
Hey @TorpedoCXC three things:

1. As these quests already have the `exclusiveTo` set, they shouldn't show up in the first place. How did you encounter an issue?
2. It is correct to use `nextQuestInChain` which then replaces the current `exclusiveTo` correction.
3. Could you try to use the quest names in the PR title instead of the IDs? Maybe even follow the format used in [previous commits](https://github.com/Questie/Questie/commits/master/)? So for this I'd prefer "[quest] Mark "The Thunderdrome!" as breadcrumb quest".
 
Side note for 3.: The PR title will become the commit message and we use the commit messages to generate the changelog for each release. If not, that is no problem, as we always have the chance to update the commit message before merging.

**Comment by TorpedoCXC** [comment]
1.) The only issue was that the alliance and horde quests "The Thunderdrome!" were not marked as breadcrumbs for "Thunderdrome: The Ginormus!". I did not encounter the issue that the breadcrumb quest still showed as available when the main quest was already done. Or to be precise: I did not test this at all.
3.) I will try to remember this. 👍 

My question on Discord was just so I learn how to mark breadcrumb quests in the future and to learn about the attributes `exclusiveTo` and `nextQuestInChain` and their differences in general.

**Comment by BreakBB** [comment]
> The only issue was that the alliance and horde quests "The Thunderdrome!" were not marked as breadcrumbs for "Thunderdrome: The Ginormus!"

Well this is strange, because both correction entries already have the `exclusiveTo` set which work very similar to `nextQuestInChain` 🤔 Once you accept "Thunderdrome: The Ginormus!" then "The Thunderdrome!" should no longer be available 🙈 It's better to use `nextQuestInChain` though 🤷🏻 👍🏻 

> I will try to remember this. 👍

Thanks! Don't stress yourself too much with it though <3

File: Database/Corrections/cataQuestFixes.lua [hunk 1]
```diff
@@ -5805,9 +5805,11 @@ function CataQuestFixes.Load()
         },
         [26895] = { -- The Thunderdrome!
             [questKeys.exclusiveTo] = {25067},
+            [questKeys.nextQuestInChain] = 25067,
```
Comment (BreakBB) [inline Database/Corrections/cataQuestFixes.lua:5808]:
```suggestion
            [questKeys.nextQuestInChain] = 25067,
```

File: Database/Corrections/cataQuestFixes.lua [hunk 2]
```diff
@@ -5805,9 +5805,11 @@ function CataQuestFixes.Load()
...
         },
         [26896] = { -- The Thunderdrome!
             [questKeys.exclusiveTo] = {25067},
+            [questKeys.nextQuestInChain] = 25067,
```
Comment (BreakBB) [inline Database/Corrections/cataQuestFixes.lua:5812]:
```suggestion
            [questKeys.nextQuestInChain] = 25067,
```

**Pull request merged**